### PR TITLE
Enable manual control of horizontal scale

### PR DESCRIPTION
**Pull request by TimSC**
The vertical axis range could already be controlled. I have added control for the horizontal scale. This addresses #20 .

### CODE REVIEW

File: Chart.Scatter.js [hunk 1]
```diff
@@ -800,7 +811,13 @@
 				lineColor: this.options.scaleLineColor,
 				display: this.options.showScale,
 
-				// range
+				// y range
```
Comment (gnclmorais) [inline Chart.Scatter.js:814]: I guess this is `x range` instead of `y range`, no?

**Comment by gnclmorais** [comment]
I just want to vouch in and thank @TimSC for this. I’m using it in a project. :trophy:

File: Chart.Scatter.js (hunk 1 above)
Comment (TimSC) [inline Chart.Scatter.js:814]: Yes, I guess it is!

**Comment by dima117** [comment]
I'm sorry for the late reply. Your PR is merged.

**Pull request merged**